vespalib/src/vespa/vespalib/stllike/hashtable.h

```diff
@@ -184,7 +185,7 @@ class hashtable : public hashtable_base
 private:
     using Node=hash_node<Value>;
 protected:
-    using NodeStore = vespalib::Array<Node>;
+    using NodeStore = std::vector<Node, allocator_large<Node>>;
```

Comment:
Consider explicitly including `<vector>` in this file as well

Reply:
There will be follow ups cleaning up includes a bit more. Will probably happen then.